### DISCUSSION TITLE
Notification template to make alerts "same as old alerting"

### DISCUSSION THREAD

**Discussion by jgournet** [grafana/grafana#68344] [Alerting]
Hi,

I'm still trying to convert to (not-so) new unified alerting.
We have a few automations behind our contact points (like opsgenie parsing grafana alerts, or even email filtering rules).

So, to make things easy, I'm going to write a notification template so that alerts look like the old one.
=> has anyone done so already via file provisioning (or terraform) ? if yes: could you share it ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
